server/config/conditions.ts

```diff
@@ -110,8 +110,8 @@ export default {
       {
         code: '5db26ab3-9b6f-4bee-b2aa-53aa3f3be7dd',
         category: 'Residence at a specific place',
-        text: 'You must reside within the [INSERT REGION] probation region while of no fixed abode, unless otherwise approved by your supervising officer.',
-        tpl: 'You must reside within the {probationRegion} probation region while of no fixed abode, unless otherwise approved by your supervising officer.',
```

Comment:
Whats the difference in putting probation region here or on the end of each option? Will we have some which dont have probation region on the end?

Reply:
The original condition text didn't have it in so we added it into the template text. This makes it part of the select list value again - but this area is, I know, being looked at again by Ricky - talk of a radio here, so can be revisited.